frontend/public/components/_dropdown.scss

```diff
@@ -260,5 +258,8 @@ $dropdown-background--hover: $color-pf-black-200; // pf-c-dropdown__menu-item--h
   .co-resource-item__resource-name {
     overflow: hidden;
     text-overflow: ellipsis;
+    .text-muted {
```

Comment:
I think we want to target this one specific case instead of hiding any muted text.

Reply:
This targets the muted text inside a button inside of a resource-name item. The outer class is a class applied to Dropdown button.

Reply:
Right, I think that's still too broad. We want to only target this one specific case. We use `co-resource-item__resource-name` in many places.

Reply:
For instance, here's a place we use it in a different dropdown:

https://github.com/TheRealJon/console/blob/62b5fec283068b665c3ade8195150762bfa84cf3/frontend/public/components/resource-dropdown.tsx#L43

Reply:
The section you mentioned wouldn't get applied this style because the parent class in not `pf-c-dropdown__toggle-text`